apps/client/src/pages/CheckerPage.js

```diff
@@ -191,15 +192,17 @@ const CheckerPage = ({ checker, topic, resetChecker }) => {
           <>
             {/* @TODO: Refactor this, because of duplicate code */}
             {isActive("locationInput") && (
-              <LocationInput {...{ topic, setActiveState, setFinishedState }} />
+              <LocationInput
+                {...{ isFinished, setActiveState, setFinishedState, topic }}
```

Comment:
Not sure what this change does. Please note the comment about Refactor this... is this code duplicated somewhere? If so, maybe also change it there?

Reply:
This is sorted correctly

Reply:
I added `isFinished` and ordered them alphabetically